mlflow/utils/databricks_utils.py

```diff
@@ -162,6 +164,15 @@ def is_in_databricks_job():
         return False
 
 
+def is_in_databricks_model_serving_environment():
```

Comment:
Should this also be an internal method?

Reply:
no, this will need to be referenced in vector search client on whether to call `get_databricks_host_creds`. Also, the other `is_in_{}`  functions are all public